gatsby/fundraisers/transformers.ts

```diff
@@ -55,8 +63,8 @@ export const createFundraiserSchemaCustomization = ({
     type DAFundraiser implements Node {
       name: String!
       title: String!
-      gallery: [DAFundraiserPhoto]!,
-      allocations: [DAFundraiserAllocation]!
+      gallery: [DAPhoto!]!
+      allocations: [DAFundraiserAllocation!]!
```

Comment:
```suggestion
      allocations: [DAFundraiserAllocation]!
```
Projects may have 0 allocations.

Reply:
@coderbyheart I think that's ok?  Based on [what I read in the docs](https://graphql.org/learn/schema/#object-types-and-fields) the `[DAFundraiserAllocation!]!` means that that graphql will always return an array with 0 or more `DAFundraiserAllocation` entries inside of it.  So it can return `[]` or `[{allocation}, ...]` but not `null` or `[{allocation}, null, ...]`

Reply:
Ah, I forgot that it means that you can have `null` for `DAFundraiserAllocation` (without `!`)